useful-scripts/misc-test-utils/src/utils/errorHandler.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable no-constant-condition */
+/* eslint-disable no-console */
+
+import { AxiosError } from 'axios';
+import chalk from 'chalk';
+
+const errorHandler = async (e: AxiosError) => {
+  switch (e!.response!.status) {
```

Comment:
Eh? Not sure why we're using non-null assertions here. We should be able to use optional chaining instead:

```ts
e.response?.status
```